ckanext/security/templates/user/snippets/login_form.html

```diff
@@ -55,6 +55,7 @@
     <p>{{_('Please enter your authenticator app generated 6-digit verification code.')}}</p>
     {{ form.input('mfa', label=_("Verification code"), id='field-mfa', type="text", value="", error=mfa_error, classes=["control-medium"], attrs={"autocomplete": "off"}) }}
 
+    <input id="mfa-form-active" name="mfa-form-active" type="hidden" value="" />
```

Comment:
Can't this be forged by an attacker who wants to grant themselves unlimited retries?